asv_bench/benchmarks/utils/common.py

```diff
@@ -504,20 +504,14 @@ def execute(
             return
 
         # compatibility with old Modin versions
-        all(
-            map(
-                lambda partition: partition.drain_call_queue() or True,
-                partitions,
-            )
-        )
         if ASV_USE_ENGINE == "ray":
             from ray import wait
 
-            all(map(lambda partition: wait([partition._data]), partitions))
+            all(map(lambda partition: wait([partition.list_of_blocks[0]]), partitions))
```

Comment:
We should wait for all blocks. Can we use the partition manager's `wait_partitions`?

Reply:
As I see, we already use internal functionality here like `df._query_compiler._modin_frame._execute()`, so I think using of `df._query_compiler._modin_frame._partition_mgr_cls.wait_partitions(partitions)` it's not against the rules ) 

Fixed

Reply:
I think that variable `ASV_USE_ENGINE` isn't need anymore, because there is no any place where it's used. But I think we should remove the definition of this var from asv_bench/benchmarks/utils/compatibility.py in a separate PR